apps/src/lab2/lab2Redux.ts

```diff
@@ -65,6 +66,9 @@ export interface LabState {
   levelProperties: LevelProperties | undefined;
   // If this lab should presented in a "share" or "play-only" view, which may hide certain UI elements.
   isShareView: boolean | undefined;
+  // User's response for the level, if the level is a predict level. It is an empty string if this
+  // is not a predict level or if the user has not yet submitted a response.
+  predictResponse: string;
```

Comment:
Yeah I also thought about this, but since it was only one field it didn't make much sense to split this out. But I think it makes sense to refactor our redux store if it gets much bigger.